apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/WebViewV2.tsx

```diff
@@ -89,6 +90,14 @@ function useUiHook(manifest: AppManifest): Partial<UiHook> {
           }),
         );
       },
+      "storage.get": async ({ key, storeId }) => {
+        const namespace = `wallet-api/${storeId}`;
+        return await getKey(namespace, key);
+      },
+      "storage.set": async ({ key, value, storeId }) => {
+        const namespace = `wallet-api/${storeId}`;
+        setKey(namespace, key, value);
+      },
```

Comment:
```suggestion
      "storage.set": ({ key, value, storeId }) => {
        const namespace = `wallet-api/${storeId}`;
        setKey(namespace, key, value);
      },
```

Reply:
get typescript errors doing this as expects return type to be `Promise<void>`